consumer/session/session_storage.go

```diff
@@ -43,9 +43,12 @@ type Storer interface {
 	GetAllFrom(bucket string, array interface{}) error
 }
 
+type timeGetter func() time.Time
```

Comment:
https://github.com/mysteriumnetwork/devbook/blob/master/golang/consumer-interfaces/README.md

Reply:
It's contract with `time.Now()` function, to mock in in tests